### PR TITLE
Hotfix v1.9.4

### PR DESCRIPTION
**Pull request by MocanaAtBroad**
Forgot to include hotfix changes from last week into v1.9.3!
These are the changes that I would like to bring over to production.

### CODE REVIEW

File: pepper-apis/config/DataDonationPlatform.yaml.ctmpl [hunk 1]
```diff
@@ -4,9 +4,11 @@ service: pepper-backend
...
 {{if eq $environment "prod"}}
-basic_scaling:
-  max_instances: 6
-  idle_timeout: 45m
+manual_scaling:
+  instances: 3
+{{else if eq $environment "staging"}}
+manual_scaling:
+  instances: 1
 {{else}}
```
Comment (MocanaAtBroad) [inline pepper-apis/config/DataDonationPlatform.yaml.ctmpl:12]: We currently using basic scaling which allows to go down to 0 instances.

File: pepper-apis/src/main/java/org/broadinstitute/ddp/monitoring/StackdriverMetricsTransmitter.java [hunk 2]
```diff
@@ -172,15 +172,10 @@ private void postQueuedMetrics(ProjectName projectName, Collection<TimeSeries> t
...
-                            LOG.warn(msg, timeSeriesList.size(), projectName.getProject(), e);
-                        } else {
-                            LOG.error(msg, timeSeriesList.size(), projectName.getProject(), e);
-                        }
+                        LOG.warn(msg, timeSeriesList.size(), projectName.getProject(), e);
                     } catch (Exception e) {
-                        LOG.error("Failed to send metrics data with {} series to project {}",
+                        LOG.warn("Failed to send metrics data with {} series to project {}",
                                 timeSeriesList.size(), projectName.getProject(), e);
                     }
```
Comment (MocanaAtBroad) [inline pepper-apis/src/main/java/org/broadinstitute/ddp/monitoring/StackdriverMetricsTransmitter.java:180]: Zim trying to hush down noise in prod-alerts channel.

File: pepper-apis/config/redisson-jcache.yaml.ctmpl [hunk 3]
```diff
@@ -4,10 +4,10 @@
...
 singleServerConfig:
   idleConnectionTimeout: 10000
-  connectTimeout: 10000
-  timeout: 6000
+  connectTimeout: 5000
+  timeout: 3000
   retryAttempts: 3
-  retryInterval: 3000
+  retryInterval: 1500
   password: null
```
Comment (MocanaAtBroad) [inline pepper-apis/config/redisson-jcache.yaml.ctmpl:11]: Trying to be a little more impatient with timeouts.

**Review by zyme — APPROVED** [review]
2nd time's the charm!

**Pull request merged**